feg/gateway/services/n7_n40_proxy/n7/model_conversion.go

```diff
@@ -287,6 +343,44 @@ func GetPolicyReauthRequestProto(
 	}
 }
 
+func GetUsageMonitoringResponsesProto(
+	updateCtx *SmPolicyUpdateReqCtx,
+	smPolicyDecision *n7_sbi.SmPolicyDecision,
+) []*protos.UsageMonitoringUpdateResponse {
+	staticRules, dynamicRules, rulesToRemove := getPccRules(smPolicyDecision)
+	responses := getUsageMonitorsProtoForUpdate(updateCtx, smPolicyDecision)
+	if len(responses) == 0 {
+		responses = append(responses, NewUsageMonitoringUpdateResponseProto(updateCtx, true))
+	}
+
+	eventTriggers, revalidationTime := getEventTriggersInfoProto(
+		smPolicyDecision.PolicyCtrlReqTriggers, smPolicyDecision.RevalidationTime)
+	firstResponse := responses[0]
```

Comment:
i think I already asked about this, but why is it 0? Why can't it have more responses?